hack/update/make-module-resources.sh

```diff
@@ -44,6 +45,25 @@ group() {
   echo -n "$(yq '.apiVersion' "$1" | awk -F '/' '/\//{print($1)}' )"
 }
 
+filterProhibitedFiles() {
+  local directory=${1}
+  if [ "$(ls -A $directory)" ]; then
+      for yaml in $directory/*
+      do
+          yq '. | select(.metadata.annotations."helm.sh/hook" == "pre-delete")' $yaml >> to-exclude.yml
+
+          yq -i '. | select(.metadata.annotations."helm.sh/hook" != "pre-delete")' $yaml
+          if [ ! -s $yaml ]; then
+              echo "Removing $yaml because of pre-delete  helm hook"
```

Comment:
```suggestion
              echo "Removing $yaml because it contained only pre-delete helm hooks"
```

Reply:
The logic excludes the file even if there is more than this hook

Reply:
so "only" word does not fit here